core/src/main/java/nl/nn/adapterframework/validation/PreparserEntityResolver.java

```diff
@@ -13,37 +13,37 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 */
-package nl.nn.adapterframework.xml;
+package nl.nn.adapterframework.validation;
 
 import java.io.IOException;
+import java.util.List;
 
-import javax.xml.transform.TransformerException;
-
+import org.apache.logging.log4j.Logger;
 import org.apache.xerces.xni.XMLResourceIdentifier;
 import org.apache.xerces.xni.XNIException;
 import org.apache.xerces.xni.parser.XMLEntityResolver;
 import org.apache.xerces.xni.parser.XMLInputSource;
 
-import nl.nn.adapterframework.core.IScopeProvider;
-import nl.nn.adapterframework.core.Resource;
+import nl.nn.adapterframework.util.LogUtil;
 
 /**
- * Xerces native EntityResolver. Appears to be only used in XercesXmlValidator currently.
+ * Xerces native EntityResolver, only used in XercesXmlValidator currently.
  * 
  * It's important that the XMLEntityResolver does not return NULL, when it cannot find a resource.
  * Returning NULL will cause the XmlReader to fall back to it's built in EntityResolver.
  * 
  * This EntityResolver can be set by using the following property on the XmlReader:
  * Constants.XERCES_PROPERTY_PREFIX + Constants.ENTITY_RESOLVER_PROPERTY
  * 
- * @author Jaco de Groot
  * @author Gerrit van Brakel
- * @see ClassLoaderURIResolver
  */
-public class ClassLoaderXmlEntityResolver extends ClassLoaderURIResolver implements XMLEntityResolver {
+public class PreparserEntityResolver implements XMLEntityResolver {
```

Comment:
Misschien dat het handig is een beschrijvende naam te gebruiken, wat doet deze preparser-entityresolver precies?